File: python/get_all_municipalities.py
from __future__ import print_function
import time
import intrinio_sdk
from intrinio_sdk.rest import ApiException
from pprint import pprint
 
intrinio_sdk.ApiClient().configuration.api_key['api_key'] = 'YOUR_API_KEY'
 
municipality_api = intrinio_sdk.MunicipalityApi()
 
next_page = '' # str | Gets the next page of data from a previous API call (optional)
 
try:
    api_response = municipality_api.get_all_municipalities(next_page=next_page)
    municipalities = api_response.municipalities

    for municipality in municipalities:
        print(municipality.government_name)
        print("-------------------------------------------------")
        print("ID:              %s" % (municipality.id))
        print("Census ID:       %s" % (municipality.census_id))
        print("Government type: %s" % (municipality.government_type))
        print("Area type:       %s" % (municipality.area_type))
        print("Area name:       %s" % (municipality.area_name))
        print("Location:        %s, %s %s" % (municipality.city, municipality.state, municipality.zip))
        print("")
except ApiException as e:
    print("Exception when calling MunicipalityApi->get_all_municipalities: %s\n" % e)

File: csharp/GetAllMunicipalities.cs
using System;
using System.Diagnostics;
using Intrinio.SDK.Api;
using Intrinio.SDK.Client;
using Intrinio.SDK.Model;

namespace Example
{
  public class GetAllMunicipalitiesExample
  {
    static void Main()
    {
      Configuration.Default.AddApiKey("api_key", "YOUR_API_KEY");

      var municipalityApi = new MunicipalityApi();
      var nextPage = "";  // string | Gets the next page of data from a previous API call (optional) 

      try
      {
        ApiResponseMunicipalities result = municipalityApi.GetAllMunicipalities(nextPage);
        result.Municipalities.ForEach(delegate (MunicipalitySummary municipality)
        {
          Console.WriteLine();
          Console.WriteLine(municipality.GovernmentName);
          Console.WriteLine("--------------------------------------");
          Console.WriteLine("ID:              " + municipality.Id);
          Console.WriteLine("Census ID:       " + municipality.CensusId);
          Console.WriteLine("Government type: " + municipality.GovernmentType);
          Console.WriteLine("Area type:       " + municipality.AreaType);
          Console.WriteLine("Area name:       " + municipality.AreaName);
          Console.WriteLine("Location:        " + municipality.City + ", " + municipality.State + " " + municipality.Zip);
        });
      }
      catch (Exception e)
      {
        Debug.Print("Exception when calling MunicipalityApi.GetAllMunicipalities: " + e.Message);
      }
    }
  }
}
